scripts/push-test-helm-charts-to-ar.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+REPO_DIR="$(readlink -f "$(dirname "$0")")/.."
+
+if [ "${GCP_PROJECT:-"unset"}" == "unset" ]; then
+  echo 'Error: GCP_PROJECT is unset. Please provide the GCP_PROJECT env variable to specify your GCP project' >&2
+  exit 1
+fi
+
+LOCATION=us
+AR_HELM_REPO=${LOCATION}-docker.pkg.dev/${GCP_PROJECT}/config-sync-test-ar-helm
+
+COREDNS_CHART=coredns
+COREDNS_CHART_VERSION=1.13.3
+COREDNS_CHART_TAR=${REPO_DIR}/e2e/testdata/${COREDNS_CHART}-${COREDNS_CHART_VERSION}.tgz
```

Comment:
Can we just store the unarchived content in git and create the archive at runtime in this script (for `coredns` and `ns-chart`)?

Reply:
DONE. One caveat is that the yaml files in the helm charts cannot have copyright header. Is that okay?

Reply:
Does helm fail to parse yaml with comments? If there's no way to work around it then we could probably exclude these files in our `lint-license-headers` script

Reply:
It can parse the yaml with comments, however, for the config not selected in the template (e.g. `{{- if .Values.serviceAccount.create }}`), it generates an empty yaml file with the copyright header.

Then Config Sync complains about a missing Kind error
```
KNV2001: Object 'Kind' is missing in 'null'

        path: /repo/source/coredns:1.13.3/coredns/templates/serviceaccount.yaml
```

Reply:
I added the charts to the `lint-license-headers`.